grabbit/src/main/groovy/com/twcable/grabbit/spring/batch/repository/JcrJobExecutionDao.groovy

```diff
@@ -341,6 +335,7 @@ class JcrJobExecutionDao extends AbstractJcrDao implements JobExecutionDao {
         jobExecution.createTime = getDate(properties[CREATE_TIME] as String)
         jobExecution.lastUpdated = getDate(properties[LAST_UPDATED] as String)
         jobExecution.version = properties[VERSION] as Integer ?: 0
+        jobExecution.transactionID = properties[TRANSACTION_ID] as Long ?: 0
```

Comment:
Trivial: Instead of `0` do `0L` so that it doesn't create an `int` that then gets cast to a `long`.